lib/formatter/csv/subject.rb

```diff
@@ -39,9 +39,8 @@ def workflow_ids
       end
 
       def locations
-        subject_locs = subject.locations.sort_by { |loc| loc.metadata["index"] }
```

Comment:
Not quite, it'll still need to return the sorted indexes when the association is loaded. Though, good point on natural sort order of the association, i'll modify it to catch any error and just revert back the natural ordering (which should be id / insertion order).